### ISSUE TITLE
No tags defined

### ISSUE THREAD

**Issue by rjbell4** [Autodesk/enterprise-config-for-git#9]
The `version.sh` now returns a version based on a tag, but there aren't any tags defined in this public repository

Suggestions:
- 361afc6d25f57b1cd40e671e3cb6be6e5bee6210 => 0.9
- 8262b9485ee084e0acd7760e38c3a24d10037ddf => 1.2.2
- 25e465e820a250da45a70fd8e924790d1a62cd15 => 1.4.2

Note that this still won't work for us, as we typically build on top of these releases.  I may change our `version.sh` to instead search for the most recent version referenced by the HEAD commit.

**Comment by rjbell4** [comment]
I think I'm going to add the tags as indicated above, and then use `git describe --always --tags --dirty` instead of the version number _and_ hash

**Comment by larsxschneider** [comment]
You are right! I must admit I neglected the open source version here a bit... just too many things to do ! Internally we are in 1.5.x already. Are you using this tool in production, too? If yes, then I would love to chat with you!

**Comment by rjbell4** [comment]
Yes, we are using it in production.  I have a few tweaks I've made that you could use, and a few that you wouldn't.  Someone else initially pulled it in, but didn't manage it particularly well, so I'm sort of taking over.  I'm completely willing to chat sometime.
